blueprints-js/component-test/qunit-rfc-232-files/__root__/__testType__/__path__/__test__.gjs

```diff
@@ -0,0 +1,36 @@
+<% if (testType === 'integration') { %>import { module, test } from 'qunit';
+import { setupRenderingTest } from '<%= modulePrefix %>/tests/helpers';
+import { render } from '@ember/test-helpers';
+import <%= componentName %> from '<%= modulePrefix %>/components/<%= componentPathName %>';
+
+module('<%= friendlyTestDescription %>', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    // Set any properties with this.set('myProperty', 'value');
```

Comment:
These comments should be updated. Using `this` in template tag tests is not recommended as the context within the component would refer to the backing class instead.

Reply:
Updated to reflect test waiter syntax in https://rfcs.emberjs.com/id/0785-remove-set-get-in-tests/